control/veda_file_ingest.py

```diff
@@ -130,55 +106,22 @@ def insert(self):
         self.complete = True
 
     def queue_job(self):
-        print '%s : [ %s ] : %s' % (
-            str(datetime.datetime.utcnow()),
-            self.video_proto.veda_id,
-            'Remote Assimilate'
-        )
-
-        '''
-        nouvelle:
-        '''
-        if self.auth_dict is None:
-            ErrorObject().print_error(
-                message='No Auth YAML Found'
+        # TODO: Break heal method listed here out into helper util
+        encode_instance = VedaHeal(
+            video_query=Video.objects.filter(
+                edx_id=self.video_proto.veda_id
             )
-            return None
-
-        # WRITE JOB QUEUEING
-        En = VedaEncode(
-            course_object=self.course_object,
-            veda_id=self.video_proto.veda_id
         )
-        self.encode_list = En.determine_encodes()
-
-        if len(self.encode_list) == 0:
-            return None
-
-        # Enqueue
-        for e in self.encode_list:
-            veda_id = self.video_proto.veda_id
-            encode_profile = e
-            jobid = uuid.uuid1().hex[0:10]
-            celeryapp.worker_task_fire.apply_async(
-                (veda_id, encode_profile, jobid),
-                queue=self.auth_dict['celery_worker_queue'].split(',')[0]
-            )
-
-        """
-        Update Video Status
-        """
-        Video.objects.filter(
-            edx_id=self.video_proto.veda_id
-        ).update(
-            video_trans_status='Queue'
+        encode_instance.send_encodes()
```

Comment:
do we need to return some value from `send_encodes` and then act accordingly here? the below log message will always be logged whatever happens in `send_encodes` method, is this correct?

Reply:
Good point. There's logging in the submethod. Deleting.